public/js/document/newsletters/sendingPanel.js

```diff
@@ -83,7 +83,7 @@ pimcore.bundle.newsletter.document.newsletters.sendingPanel = Class.create({
             for(var i = 0; i < adapterNames.length; i++) {
                 //skip adapters that require a namespace that is not available
                 let requiredNamespace = pimcore.bundle.newsletter.document.newsletters.addressSourceAdapters[adapterNames[i]].prototype.requiredNamespace;
-                if (requiredNamespace && typeof pimcore.bundle[requiredNamespace]['startup'] === 'undefined') {
+                if (requiredNamespace && (typeof pimcore.bundle[requiredNamespace] === 'undefined' || typeof pimcore.bundle[requiredNamespace]['startup'] === 'undefined')) {
```

Comment:
```suggestion
                if (requiredNamespace && typeof pimcore.bundle[requiredNamespace]?.startup === 'undefined') {
```
should be easier to read